lib/unhangout-routes.js

```diff
@@ -788,6 +795,136 @@ module.exports = {
         // Starting and stopping events
         //
 
+        app.get("/followup/event/:id/participant_:participantCount", function(req, res) {
```

Comment:
We also need access control on this view. It's not enough to just hide the link; middleware should be used to restrict access to this view to superusers (e.g. ``utils.ensureSuperuser``).